openssl/src/dsa.rs

```diff
@@ -25,6 +64,12 @@ impl DsaRef {
     private_key_to_der!(ffi::i2d_DSAPrivateKey);
     public_key_to_der!(ffi::i2d_DSAPublicKey);
 
+    /// Returns the size of the signature output by `self` in bytes.  Returns
```

Comment:
It specifically returns the maximum size since the actual signature size can vary.